src/main/java/uk/gov/hmcts/reform/notificationservice/config/QueueClientConfig.java

```diff
@@ -4,34 +4,40 @@
 import com.microsoft.azure.servicebus.IMessageReceiver;
 import com.microsoft.azure.servicebus.ReceiveMode;
 import com.microsoft.azure.servicebus.primitives.ServiceBusException;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
 import org.springframework.beans.factory.annotation.Value;
 import org.springframework.boot.autoconfigure.condition.ConditionalOnProperty;
 import org.springframework.context.annotation.Bean;
 import org.springframework.context.annotation.Configuration;
 
+import java.util.Arrays;
 
 @Configuration
 public class QueueClientConfig {
 
+    private static final Logger log = LoggerFactory.getLogger(QueueClientConfig.class);
+
     @Bean
     @ConditionalOnProperty(name = "queue.notifications.read-connection-string")
     public IMessageReceiver notificationsMessageReceiver(
         @Value("${queue.notifications.read-connection-string}") String connectionString)
         throws InterruptedException, ServiceBusException {
+        logQueueName(connectionString);
         return ClientFactory.createMessageReceiverFromConnectionString(
             connectionString,
             ReceiveMode.PEEKLOCK
         );
     }
 
-    @Bean
-    @ConditionalOnProperty(name = "queue.notifications.staging-enabled")
-    public IMessageReceiver notificationsTestMessageReceiver(
-        @Value("${queue.notifications.staging-read-connection-string}") String connectionString)
-        throws InterruptedException, ServiceBusException {
-        return ClientFactory.createMessageReceiverFromConnectionString(
-            connectionString,
-            ReceiveMode.PEEKLOCK
-        );
+    private void logQueueName(String connectionString) {
+        String[] split = connectionString.split(";");
+        Arrays.stream(split)
+            .filter(e -> e.startsWith("EntityPath"))
+            .findFirst()
+            .ifPresent(c ->
+                log.warn("Notification service connected to queue: {} ", c)
```

Comment:
perhaps need to strip the "prefix" of `EntityPath` - just need to inform which queue it is connected to.

oh, and does it suppose to be warning? :monocle_face:

Reply:
as it is important and it will print once , Warning might be better.

Reply:
EntityPath  removed from log.